test/bn_spatial_test.cpp

```diff
@@ -1130,6 +1130,8 @@ struct batch_norm_spatial_driver : test_driver
     batch_norm_spatial_driver()
     {
         this->batch_factor = 4;
+        this->tolerance =
+            4e-3 / std::numeric_limits<T>::epsilon(); // ck solver has tolerance of 4e-3

```

Comment:
That is actually `33554.4336f` for float and `18014398509481.984` for double, is it intended?

Reply:
Since CK kernel have tolerance of 4e-3. Should I just set the tolerance to that?

Reply:
By the way, I'm not sure that it's an appropriate place for `tolerance` initialization.
For example it is overridden here:
https://github.com/ROCm/MIOpen/blob/42d6e9f4ba44400d0b2164265a2b843b46834334/test/bn_3d_spatial_test.cpp#L1372-L1373

Reply:
I will directly initialize there then for bn_spatial_test.cpp and bn_3d_spatial_test.cpp